Clean/Table/Basic.lean

```diff
@@ -402,6 +434,22 @@ instance [Repr F] : Repr (TableOperation S F) where
     | .EveryRow c => "EveryRow " ++ reprStr c
     | .EveryRowExceptLast c => "EveryRowExceptLast " ++ reprStr c
 
+instance : ToJson (TableOperation S F) where
+  toJson op := match op with
+    | .Boundary i c => Json.mkObj [
+        ("Boundary", Json.mkObj [
+          ("row", toJson i),
+          ("context", toJson c)
+        ])
```

Comment:
I think using the type of table operation (e.g. "Boundary") as a key is not optimal, usually enums are easiest to deal with by introducing a `"type"` key for them.

```suggestion
    | .Boundary i c => Json.mkObj [
      ("type", Json.str "Boundary"),
      ("row", toJson i),
      ("context", toJson c)
    ]
```